plugins/orion/src/components/workflow/topology/DemoTaskNode.tsx

```diff
@@ -75,16 +64,12 @@ const DemoTaskNode: any = ({
       <TaskNode
         ref={hoverRef}
         element={element}
-        onContextMenu={data.showContextMenu ? onContextMenu : undefined}
-        contextMenuOpen={contextMenuOpen}
-        scaleNode={
-          (hover || contextMenuOpen) && detailsLevel !== ScaleDetailsLevel.high
-        }
+        // onContextMenu={data.showContextMenu ? onContextMenu : undefined}
```

Comment:
do we need to keep this commented out code or can it be removed?

Reply:
I will remove them